serveradmin/servershell/static/js/servershell/autocomplete/search.js

```diff
@@ -50,8 +50,6 @@ $(document).ready(function () {
         delay: 250, // Wait n ms before starting to auto complete to avoid needles requests to backend
         autoFocus: true,
         source: function (request, response) {
-            spinner.enable();
```

Comment:
Comment for the whole commit:
Looks like it would make sense in future to implement something more global to use, something like:
``` js
Serveradmin.sendCommand(...) // sendRequest // request // or the like
```
Which would then do the calls to the spinner, so you don't need to put it to every single space.

Reply:
I thought about this but in some places we do $.ajax and in some $.getJSON and settings vary greatly so if there is not method to hook into the XHR requests I don't think it is worth it.

Reply:
With a appropriate service class/object we could still achieve it. But that's something for another time. ;)